src/utils/particle/bubbler.ts

```diff
@@ -4,40 +4,44 @@ import { Utils } from "../utils";
 import { HoverMode, ClickMode, ProcessBubbleType } from "../enums";
 
 export class Bubbler {
-    private readonly particle: Particle;
-    private readonly container: Container;
-
     public opacity?: number;
     public radius?: number;
 
+    private readonly particle: Particle;
+    private readonly container: Container;
+
     constructor(container: Container, particle: Particle) {
         this.container = container;
         this.particle = particle;
     }
 
-    private init() {
-        const particle = this.particle;
-
-        this.opacity = particle.opacity.value;
-        this.radius = particle.radius;
-    }
-
-    public bubble() {
+    public bubble(): void {
         const container = this.container;
         const options = container.options;
+        const hoverEnabled = options.interactivity.events.onhover.enable;
+        const hoverMode = options.interactivity.events.onhover.mode;
+        const clickEnabled = options.interactivity.events.onclick.enable;
+        const clickMode = options.interactivity.events.onclick.mode;
 
         /* on hover event */
-        if (options.interactivity.events.onhover.enable && Utils.isInArray(HoverMode.bubble, options.interactivity.events.onhover.mode)) {
+        if (hoverEnabled && Utils.isInArray(HoverMode.bubble, hoverMode)) {
             this.hoverBubble();
-        } else if (options.interactivity.events.onclick.enable && Utils.isInArray(ClickMode.bubble, options.interactivity.events.onclick.mode)) {
+        } else if (clickEnabled && Utils.isInArray(ClickMode.bubble, clickMode)) {
             this.clickBubble();
         }
     }
 
-    public process(dist_mouse: number, time_spent: number, bubble_param: number, particles_param: number, p_obj_bubble: number | undefined, p_obj: number, id: ProcessBubbleType) {
+    private init(): void {
+        const particle = this.particle;
+
+        this.opacity = particle.opacity.value;
+        this.radius = particle.radius;
+    }
+
+    private process(dist_mouse: number, time_spent: number, bubble_param: number, particles_param: number, p_obj_bubble: number | undefined, p_obj: number, id: ProcessBubbleType): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)